_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,83 @@
+
+[^4]: How exactly does the randomness produce this broad distribution? In a nutshell, it happens because there are multiple random events all happening together. A random subset of bacteria acquire spacers at random times, and then the processes of bacterial growth and death (also random) magnify the initial randomness. Imagine two different spacer acquisition events: if one happens a little earlier than the other, the first bacterium has more time to produce offspring that also contain the spacer, and since they're growing exponentially, that initial head start can mean a large difference in abundance later. Mathematically, we found that these distributions were most broad if the spacer acquisition probability was low (but still the same for every spacer). If spacer acquisition rates are high across the board, the randomness of bacterial growth and death gets 'washed out' by many acquisitions of the same sequence, and the abundance distribution becomes more sharply peaked. The gory details are in the supplementary material of [our paper](https://www.pnas.org/content/115/32/E7462) ([direct PDF link](http://madeleinebonsma.com/wp-content/uploads/2018/07/Bonsma-Fisher_PNAS_2018.pdf)).
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/banfield_data_theory.svg" alt="banfield_data_theory" width="800px" style="padding:5px;">
+</p>
+
+Does this mean that the study that found high correlation between replicates was wrong to conclude that acquisition was biased? Not at all, in fact their study demonstrates that beyond a doubt! But here's the very important catch: our work shows that if you observe a broad distribution by itself, you *cannot* conclude that biased acquisition is the cause, because we know you don't need that ingredient to get a similar result. These things are all true at once: spacer acquisition IS biased, biased acquisition CAN lead to broad abundance distributions, BUT observing a broad abundance distribution does not guarantee that acquisition is biased. 
+
+Why does this matter? In lab experiments like those described here, you can do multiple copies of the same experiment under the same conditions to prove a result. But the world is full of important and fascinating research areas where that isn't possible. Any study of bacteria in a natural environment is inherently unreproducible: you can never turn back the clock and do things the same way twice. So you trek to the glacial meltwater lake, sequence millions of bacteria, find all their CRISPR spacers, and plot the abundance distribution. How can you figure out what's going on? In biology *everything* is going on, and the key to understanding is to figure out *which* of the things going on are important for what you observe and which are not. And our null model taught us that if you want to learn about spacer acquisition, you'll need to look beyond a broad spacer abundance distribution to find it. 
```

Comment:
You can take out the “And” at the begining of the last sentence. 

Great story and great blog! Made me want to read the paper (which I will do now haha). Remember that if you disagree with my comments or want to add something you can just reply to them and we can do another round. Reviewers are often wrong!

Reply:
Thanks for the great comments! Just pushed some changes.